modules/ROOT/pages/simulating-api-calls-using-the-mocking-service.adoc

```diff
@@ -3,20 +3,20 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Using the mocking service in the API Console of the portal, you can simulate calls to your API. The mocking service is a feature of Anypoint that enables you to generate simulated servers that aid in the development and testing of APIs. These mock servers imitate the actions of accepting requests and providing responses. The mocking service replicates the functionality of a genuine API. You can run the mocking service from the text editor, the visual editor, and from Anypoint Exchange.
+Using the mocking service in the API Console of the portal, you can simulate calls to your API. The mocking service is a feature of Anypoint that enables you to generate simulated servers that aid in the development and testing of APIs. These mock servers imitate the actions of accepting requests and providing responses. The mocking service replicates the functionality of a genuine API. 
 
 == Before You Begin
 
 You must have an API specification with at least one endpoint and one method in it.
 
 == Simulate a Call to an API Using an Internal URL
-To access the internal URL, you must specify the authorization headers as part of the request URL. To test an API without supplying authorization headers, generate a mocking service public URL for this project. Mocking service is the selected server in the *Try it* panel. 
+To access the internal URL, you must specify the authorization headers as part of the request URL. To test an API without supplying authorization headers, generate a mocking service public URL for this project. Mocking service is the selected server in the *Try it* pane. 
```

Comment:
It shouldn't be panel instead of pane

Reply:
Hi Alex, Thanks for pointing that out. Since I'm accustomed to using the word "pane" in the Exchange documentation, I'm also using "pane" consistently in AEH as well.